nimare/decode/discrete.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Methods for decoding subsets of voxels (e.g., ROIs) or experiments (e.g., from
+meta-analytic clustering on a database) into text.
+"""
+from .base import Decoder
+from ...due import due, Doi
+
+
+@due.dcite(Doi('10.1371/journal.pcbi.1005649'),
+           description='Citation for GCLDA decoding.')
+class GCLDADiscreteDecoder(Decoder):
```

Comment:
Is it worth it for me to try to fit GCLDA (including the dataset and model modules) into NiMARE or should I just try to make GCLDA NiMARE-compatible after NiMARE is functional?